File: web/js/shinow/app/model/OrderInfo.js
/**
 * <p>
 * 订单信息表
 * </p>
 * 
 * @author 代码生成器
 * @class app.model.OrderInfo
 * @extends Ext.data.Model
 */
Ext.define('app.model.OrderInfo', {
		extend : 'Ext.data.Model',
		alias : 'model.OrderInfo',
		idProperty:'billCode',
		fields : [
				{
					name : 'billCode',
					type: 'int'
				},
				{
					name : 'deliveryCode',
					type: 'string'
				},
				{
					name : 'outStockId',
					type: 'int'
				},
				{
					name : 'userName',
					type: 'string'
				},
				{
					name : 'userId',
					type: 'int'
				},
				{
					name : 'postBillCode',
					type: 'string'
				},
				{
					name : 'billStatus',
					type: 'int'
				},
				{
					name : 'orderTime',
					type: 'date'
				},
				{
					name : 'recvName',
					type: 'string'
				},
				{
					name : 'linkTel',
					type: 'string'
				},
				{
					name : 'recvAddress',
					type: 'string'
				},
				{
					name : 'postCode',
					type: 'string'
				},
				{
					name : 'totalMoney',
					type: 'string'
				},
				{
					name : 'remark',
					type: 'string'
				}
		]
});

File: src/main/java/com/shinowit/entity/OrderInfo.java
package com.shinowit.entity;

import java.io.Serializable;
import java.util.Date;

public class OrderInfo implements Serializable {
    private Integer billCode;

    private String deliveryCode;

    private Integer outStockId;

    private String userName;

    private Integer userId;

    private String postBillCode;

    private Byte billStatus;

    private Date orderTime;

    private String recvName;

    private String linkTel;

    private String recvAddress;

    private String postCode;

    private Float totalMoney;

    private String remark;

    private static final long serialVersionUID = 1L;

    public Integer getBillCode() {
        return billCode;
    }

    public void setBillCode(Integer billCode) {
        this.billCode = billCode;
    }

    public String getDeliveryCode() {
        return deliveryCode;
    }

    public void setDeliveryCode(String deliveryCode) {
        this.deliveryCode = deliveryCode;
    }

    public Integer getOutStockId() {
        return outStockId;
    }

    public void setOutStockId(Integer outStockId) {
        this.outStockId = outStockId;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public String getPostBillCode() {
        return postBillCode;
    }

    public void setPostBillCode(String postBillCode) {
        this.postBillCode = postBillCode;
    }

    public Byte getBillStatus() {
        return billStatus;
    }

    public void setBillStatus(Byte billStatus) {
        this.billStatus = billStatus;
    }

    public Date getOrderTime() {
        return orderTime;
    }

    public void setOrderTime(Date orderTime) {
        this.orderTime = orderTime;
    }

    public String getRecvName() {
        return recvName;
    }

    public void setRecvName(String recvName) {
        this.recvName = recvName;
    }

    public String getLinkTel() {
        return linkTel;
    }

    public void setLinkTel(String linkTel) {
        this.linkTel = linkTel;
    }

    public String getRecvAddress() {
        return recvAddress;
    }

    public void setRecvAddress(String recvAddress) {
        this.recvAddress = recvAddress;
    }

    public String getPostCode() {
        return postCode;
    }

    public void setPostCode(String postCode) {
        this.postCode = postCode;
    }

    public Float getTotalMoney() {
        return totalMoney;
    }

    public void setTotalMoney(Float totalMoney) {
        this.totalMoney = totalMoney;
    }

    public String getRemark() {
        return remark;
    }

    public void setRemark(String remark) {
        this.remark = remark;
    }
}